.github/workflows/screeners.yml

```
@@ -0,0 +1,217 @@
+name: Screeners workflow
+
+on: pull_request
+
+env:
+        SKIPCOMPONENTGOVERNANCEDETECTION: true
+
+        ConnectionString: ${{ secrets.AZURE_STORAGE_CONNECTION_STRING}}
+
+        BROWSERSLIST_IGNORE_OLD_DATA: true
+
+        DEPLOYHOST: 'fluentuipr.z22.web.core.windows.net'
+
+        SYSTEM_PULLREQUEST_TARGETBRANCH: ${{ github.event.pull_request.base.ref }}
+
+        SYSTEM_PULLREQUEST_SOURCEBRANCH: ${{ github.event.pull_request.head.ref }}
+
+        BUILD_SOURCEBRANCHNAME: 'merge'
```

```suggestion
        BUILD_SOURCEBRANCHNAME: 'merge'
```

This will break for `master` builds since the source branch name should be master